manifests/prepare/package.pp

```diff
@@ -33,15 +33,17 @@
 
     if $::osfamily =~ /windows/ {
       $local_package_file_path = windows_native_path("${::puppet_agent::params::local_packages_dir}/${package_file_name}")
+      $mode = '0664'
```

Comment:
Is there a reason we even need to set a mode on Windows?

Would `undef` be better? (I have not verified that as viable)